tools/isopack.js

```diff
@@ -1117,7 +1117,8 @@ _.extend(Isopack.prototype, {
           // If we're going to write a legacy prelink file later, track the
           // original form of the resource object (with the source in a Buffer,
           // etc) instead of the later version.  #HardcodeJs
-          if (writeLegacyBuilds && resource.type === "source" &&
+          if (writeLegacyBuilds &&
```

Comment:
Hmm, it seems like we need to call `compiler.getActivePluginPackages` to determine if the current isopack is processed by any plugins that register compiler handlers for `.js` files, but it's too early to call `compiler.getActivePluginPackages` here, so that optimization may not be available?